### PR TITLE
21 student profile

### PR DESCRIPTION
**Pull request by AlaaTaima**
Relates #21

### CODE REVIEW

File: client/src/components/pages/StudentProfile/index.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.schoolName {
+    padding: 80px 15px 0 15px;
+  }
+  .schoolName__tiltle {
```
Comment (aminalakhsham) [inline client/src/components/pages/StudentProfile/index.css]: add a line inline 4
and fix indentation from 4 to 9
```suggestion
.schoolName__tiltle {
```

Comment (AlaaTaima) [reply client/src/components/pages/StudentProfile/index.css]: ok , I will edit it

File: client/src/components/common/TeacherTable/index.css [hunk 2]
```diff
@@ -1,45 +1,50 @@
...
+  font-size: 0.9rem;
+  font-weight: bold;
+  background-color: #528af5;
+}
+
 .studentTable__title2 {
-    border-bottom-right-radius: 0.5rem;
-    border-top-right-radius: 0.5rem;
-} 
+  border-bottom-right-radius: 0.5rem;
```
Comment (aminalakhsham) [inline client/src/components/common/TeacherTable/index.css]: make border in one line

Comment (AlaaTaima) [reply client/src/components/common/TeacherTable/index.css]: ok I will edit them.

File: client/src/components/common/TeacherTable/index.js [hunk 3]
```diff
@@ -1,45 +1,60 @@
...
+        ) : (
+          <table className="studentTable">
+            <thead className="studentTable__title">
+              <tr>
+                <th className="studentTable__title1">Teacher Name</th>
+                <th className="studentTable__title2">Subject</th>
+              </tr>
+            </thead>
+            <tbody className="studentTable__content">
+              {data.map(({ teacherName, subject, id }) => (
```
Comment (Mohammed-Q96) [inline client/src/components/common/TeacherTable/index.js]: - [ ] when using map in react you must use key props 

https://reactjs.org/docs/lists-and-keys.html#basic-list-component

File: client/src/components/common/TeacherTable/index.css [hunk 4]
```diff
@@ -1,45 +1,46 @@
 .studentTable {
```
Comment (asem1789) [inline client/src/components/common/TeacherTable/index.css]: I think we shouldn't use CamelCase in css
.student-table

most of classes name camelCase

**Pull request merged**